pkg/util/metric/metric.go

```diff
@@ -17,18 +17,28 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/cockroachdb/cockroach/pkg/util"
+	"github.com/cockroachdb/cockroach/pkg/util/envutil"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/codahale/hdrhistogram"
 	"github.com/gogo/protobuf/proto"
 	"github.com/prometheus/client_golang/prometheus"
 	prometheusgo "github.com/prometheus/client_model/go"
 	"github.com/rcrowley/go-metrics"
 )
 
 const (
+	// MaxLatency is the maximum value tracked in latency histograms. Higher
```

Comment:
This only applies to `hdrhistograms` - do we want to separate the hdrhistogram code from the current code by moving it into a separate file?

I'm going to assume all constants like these were brought back via a kind of (manual or real) revert.

Reply:
This was done via a manual revert/rework of https://github.com/cockroachdb/cockroach/commit/a82aa8267b1b33a685a02a0f73ec2ece1f8ba295

I've moved all the hdrhistogram code into a separate file as suggested.